x-pack/plugins/watcher/public/index.ts

```diff
@@ -3,6 +3,8 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+import './index.scss';
```

Comment:
Just double-checking, but importing styles in the plugin's index file will always put those styles in the head of the current page (no matter what that page is in Kibana)? 

For instance, this index file contains the `.watcherThresholdAlertAggFieldContainer` selector. If I'm in Discover, I will find this selector and it's styles in the `<head>` of the page. However, if the plugin author, moves the styles import to the component level, the selector will only get injected to the `<head>` if the component exists on the page?

Reply:
Yes, sort of, the styles are injected into the page whenever the module is loaded, which is on page bootstrap for plugin index files and might be on bootstrap for components too, unless they're behind an `await import()` somewhere. In that case the styles will be loaded into the `<head>` of the page the first time the `await import()` is called and they will stay there until the page is refreshed.

Reply:
Ah ok. We'll probably eventually want some more detailed docs too for when/where it's most appropriate to load styles. But this definitely gets the platform in place. 🙇‍♀